checker-qual/src/main/java/org/checkerframework/checker/regex/qual/PolyRegex.java

```diff
@@ -11,9 +11,11 @@
 /**
  * A polymorphic qualifier for the Regex type system.
  *
- * <p>Any method written using {@link PolyRegex} conceptually has two versions: one in which every
- * instance of {@link PolyRegex} has been replaced by {@link Regex}, and one in which every instance
- * of {@link PolyRegex} has been replaced by {@link UnknownRegex}.
+ * <p>Any method written using {@link PolyRegex} has multiple versions, every instance of {@link
+ * PolyRegex} can be replaced by one of following qualifier: {@link Regex}, which can take an
+ * integer argument to represent different capturing groups; {@link PartialRegex}, which can take
+ * string argument to represent different partial regexes; {@link UnknownRegex} and {@link
+ * RegexBottom}.
```

Comment:
```suggestion
 * <p>Any method written using {@link PolyRegex} conceptually has multiple versions: one in which all instances of {@link
 * PolyRegex} in a method signature have been replaced by one of the following qualifiers: {@link Regex}, which takes an
 * integer argument to represent different capturing groups; {@link PartialRegex}, which takes
 * a string argument to represent different partial regexes; {@link UnknownRegex}; and {@link
 * RegexBottom}.
```

Reply:
You will probably need to reformat the text. Please make similar changes in #494 .

Reply:
Thanks, the only thing I changed now is in `the` method as we discussed in the morning.